core/chains/evm/log/pool.go

```diff
@@ -9,12 +9,44 @@ import (
 	pairingHeap "github.com/theodesp/go-heaps/pairing"
 )
 
+//go:generate mockery --name iLogPool --output ./ --inpackage --testonly
+
+// The Log Pool interface.
+type iLogPool interface {
```

Comment:
I think it would be OK to export here since it may eventually be necessary anyways, and to follow simpler naming conventions:
```suggestion
type LogPool interface {
```

Reply:
But wouldn't that export this outside of this package?
Or are you saying we eventually need to export this outside of the package?

Reply:
I don't understand why this is all public now. It seems like we've just exposed all these methods publicly which don't need to be, isn't this the opposite of what we're trying to achieve?

Reply:
I didn't make it public.
I put the name as iLogPool, so it is still unexported.
I think @jmank88 suggested to change it to LogPool.

My preference is to keep it private and not export it, just for testability.

Reply:
Right but all the methods are public?

Reply:
Not exporting anything as per latest discussions